docs/source/concepts/lifecycle.rst

```diff
@@ -1,22 +1,148 @@
+.. _lifecycle_concept:
+
+========================
 The Simulation Lifecycle
 ========================
 
+A ``vivarium`` simulation has many important lifecycle phases.
+
+It begins with a user interacting with one of the simulation
+:ref:`entry points <entry_points_concept>`. The simulation then goes through an
+initialization cycle that creates all the :term:`components <Component>`,
+and compiles :term:`configuration information <Configuration>`.  After
+initialization, the simulation enters the setup phase, where each component
+in turn has a special ``setup`` method called with the simulation
+:ref:`builder <builder_concept>`.  During the setup, components ask for
+services from the framework and register themselves with various simulation
+subsystems.  There is then a post setup phase where the simulation framework
+does some cleanup.
+
+The simulation then enters its main processing section. The population
+is initialized, then the time step loop begins. Once the loop completes,
+results are calculated, gathered, and written out.
+
 .. contents::
    :depth: 2
    :local:
    :backlinks: none
 
-Outline
--------
-
-- Entry point variations (link to more detailed guide)
-- Initialization (create objects, compile configuration)
-- Setup (Wire components, load data)
-- Post setup
-- Population Initialization
-- Event loop
-  - time_step__prepare
-  - time_step
-  - time_step__cleanup
-  - collect_metrics
-- simulation_end
+Initialization
+--------------
+
+The initialization phase of ``vivarium`` lasts from when a user first interacts
+with a simulation :ref:`entry point <entry_points_concept>` to when the
+``__init__`` method of the :class:`vivarium.framework.engine.SimulationContext`
+completes.  In it, the following things happen (roughly in this order):
+
+1. Simulation outputs are configured. This is usually just setting up some
+   directories for the final results and the simulation logs.
+2. :term:`Model specification <Model Specification>` defaults are collected and
```

Comment:
Is it worth clarifying here that only the configuration defaults + configuration block of the model spec file are compiled into the ConfigTree not the whole model spec?

Reply:
I like the clarification that component defaults aren't compiled here. But I'm still a little confused - maybe I'm misreading this but if the user doesn't supply a model specification file, where are the 'Model specification defaults' collected from?

Reply:
these come from the user's ``vivarium.yaml`` file if present.  I'll write more about it in the ``Configuration`` concept section and link that here when done.

Reply:
oh I forgot about `vivarium.yaml`